friends/src/components/FriendAdd/FriendAdd.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const FriendAdd = props => {
+	console.log(props);
+	return (
+		<form onSubmit={props.addFriend} className="friendAdd">
+			Name:{" "}
```

Comment:
```suggestion
			<label htmlFor="name">Name</label>
```
You need a label tag for this. The htmlFor property should match the id on the input associated with it, so you'll have to set and id of name of your input as well.